### DISCUSSION TITLE
［委托］众筹280w丁尼或者摩拉或者信用点或者星石给我家猫买冻干

### DISCUSSION THREAD

**Discussion by abyssal-sea-c** [share121/inter-knot#5445] [委托]
我家猫因为吃不上冻干都给饿瘦了😭快给我捐款
![IMG_20240928_082211](https://github.com/user-attachments/assets/56c238ca-f751-4355-9363-8cb51c21fb50)
![IMG_20240928_071605](https://github.com/user-attachments/assets/3a055d71-87d2-45f7-bd07-327ee78674e1)
![IMG_20240928_082217](https://github.com/user-attachments/assets/5e06796b-7164-48a9-90d2-04335c6949bb)

**Comment by wshtdgo** [comment]
我有1000多w信用点and500多w摩拉（忍痛递）各给你280w，谁叫小猫这么可爱呢（条件给我撸两天）（星星眼）

**Reply by abyssal-sea-c** [reply]
好的喵 在你做梦的时候猫猫会出现在你的梦里喵（丁尼+280w）

**Comment by KELERSAMA** [comment]
我也想要小猫牌的电风扇 
<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >

**Reply by abyssal-sea-c** [reply]
我也想要

**Comment by SteveNotA** [comment]
猫猫！

**Comment by SteveNotA** [comment]
猫猫！

**Reply by abyssal-sea-c** [reply]
猫猫
